mssqlcli/jsonrpc/contracts/connectionservice.py

```diff
@@ -144,38 +143,3 @@ class ConnectionResponse(object):
     def __init__(self, params):
         self.result = params[u'result']
         self.id = params[u'id']
-
-
-#
-#   Handle Connection Events.
-#
-
-def handle_connection_response(response):
-    # Handles complete notification and return ConnectionCompleteEvent object
-    # if connection successful.
-    def handle_connection_complete_notification(response):
-        if not response.connection_id:
-            sys.stderr.write(u'\nConnection did not succeed.')
-            if response.error_message:
-                sys.stderr.write(u'\nError message: ' + response.error_message)
-                logger.error(response.error_message)
-            if response.messages:
-                logger.error(response.messages)
-
-        return response
-
-    def handle_connection_response_notification(response):
-        if not response.result:
-            sys.stderr.write(
-                u'\nIncorrect json rpc request. Connection not successful.')
-        return None
-
```

Comment:
These output to the terminal connection error messages. Without these the user would'nt know why the connection did not succeed.

Reply:
Sure, I can surface error messages for the client to output to the console. For this particular error how do we know the lack of a result means the json rpc request was invalid? also this message doesn't seem to be actionable for the user since we are forming the request.

Reply:
We would get back a result if toolsservice was able to understand our request. If we don't have a result parameter it means something went wrong in jsonrpc request. 

---
In reply to: [168625939](https://github.com/dbcli/mssql-cli/pull/169#discussion_r168625939) [](ancestors = 168625939)